internal/agent/device/spec/spec_test.go

```diff
@@ -94,7 +94,7 @@ func TestManager(t *testing.T) {
 				return
 			}
 			if tt.wantErr != nil {
-				require.ErrorAs(err, &tt.wantErr)
+				require.ErrorAs(err, &tests[tIndex].wantErr)
```

Comment:
`tt` does this for you

Reply:
There was a lint warning about passing a pointer to the loop variable

Reply:
I think we just want ErrorIs so we don't need a ref

Reply:
Right, removed the ref.